tests/installation/select_patterns_and_packages.pm

```diff
@@ -60,6 +60,11 @@ sub gotopatterns {
         send_key 'alt-s';
     }
     else {
+        # See poo#12322. Prevent tab presses before overview is fully loaded
+        if (!check_screen('installation-settings-overview-loaded')) {
```

Comment:
That's why messages says "may". Do you suggest to switch to assert_screen then? If Overview is not loaded at all, it will fail on previous test case, here we select packages. We can move this part to installation_overview_before and assert_screen there.